.gitignore

```diff
@@ -2,7 +2,7 @@
 /dist/
 /.idea/
 /.vscode/
-nbproject/private/
+/nbproject/
```

Comment:
I believe the existing nbproject/private/ is meant to catch any private files created by someone opening any of the various nbproject directories that exist in the source base, so it should probably stay. Adding an ignore for any top level nbproject directory in addition to that should be ok.

Reply:
Thanks @erikj79 for your review.
Updated.

Any comments?
Thanks.

Reply:
You misunderstand me. Adding the following line is fine and will make git ignore any nbproject dir in the root of the repository, which is what I assume you want:
/nbproject/

But removing the following existing line is not, as this matches any potential private part of already existing nbproject directories:
nbproject/private/

Reply:
Adding the below will make git ignore a bunch of already existing nbproject files.
nbproject/configurations.xml
nbproject/project.xml

Reply:
Got it.
Thanks @erikj79 for your kind explanation.
Updated.